pkgs/development/libraries/libxslt/default.nix

```diff
@@ -3,9 +3,6 @@
 , pythonSupport ? stdenv.buildPlatform == stdenv.hostPlatform
 }:
 
-assert pythonSupport -> python != null;
```

Comment:
I *think*, if I've understood correctly in my reading of the linked issues, that the reason for deleting this line is that it actually wasn't doing anything at all, because `python` will never be `null`. Is that right? Or was this line also causing problems of its own?

Reply:
It's being removed because it prevents one from overriding the expression.

Reply:
This particular line does not prevent overriding. In principle we can keep it, but I would like to know what problem this assertion prevents.

Reply:
Actually, I am not correct. It does prevent a deep override:
```
with import ./. {
  config = {
    packageOverrides = pkgs: {
      python2 = null;
      python3 = null;
      libxslt = pkgs.libxslt.override { pythonSupport = false; };
    };
  };
};

libxslt
```

And I actually use those myself sometimes.